plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/ViperPoweredDeclarationChecker.kt

```diff
@@ -42,11 +42,10 @@ class ViperPoweredDeclarationChecker(private val session: FirSession, private va
     override fun check(declaration: FirSimpleFunction, context: CheckerContext, reporter: DiagnosticReporter) {
         if (!config.shouldConvert(declaration)) return
         val errorCollector = ErrorCollector()
-        var program: Program? = null
         try {
             val programConversionContext = ProgramConverter(session, config, errorCollector)
             programConversionContext.registerForVerification(declaration)
-            program = programConversionContext.program
+            val program = programConversionContext.program
```

Comment:
Is there any value to having it as a local anyway, or can you pass `programConversionContext.program` directly to `getProgramForLogging`?